### PR TITLE
Amend trend docs with config flow details

### PR DESCRIPTION
**Pull request by boswelja**
## Proposed change

Updated docs to match changes in https://github.com/home-assistant/core/pull/80408

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/80408
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by boswelja** [comment]
It looks like the documentation here is relatively outdated and doesn't really conform to the current standard? I'm happy to spend some time updating it, though not sure if it should be a separate PR from this.

**Comment by boswelja** [comment]
My bad, I forgot to mark as ready for review

File: source/_integrations/trend.markdown [hunk 1]
```diff
@@ -19,6 +21,8 @@ at least two updates of the underlying sensor to establish a trend.
 Thus it can take some time to show an accurate state. It can be useful
 as part of automations, where you want to base an action on a trend.
 
+{% include integrations/config_flow.md %}
+
 ## Configuration
```
Comment (frenck) [inline source/_integrations/trend.markdown]: The config flow include will have a heading called "Configuration" as well. We should make clear that this helper can be configured both via the UI as helper and manually via YAML.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/trend.markdown (hunk 1 above)
Comment (boswelja) [reply source/_integrations/trend.markdown]: I have made some changes, what do you think?

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request closed without merging**